packages/@dcos/sorting/index.ts

```diff
@@ -0,0 +1,65 @@
+export enum ComparisonResult {
+  BIGGER = 1,
```

Comment:
Shall we stick with a well known terms like `LESS_THAN` and `GREATER_THAN`?  Or even `LT` `GT` https://en.wikipedia.org/wiki/Less-than_sign